CMakeLists.txt

```diff
@@ -107,13 +107,13 @@ if(DEBUG_SYMBOL)
     endif()
 endif()
 
-option(MODMESH_PROFILE "enable profiler" OFF)
-message(STATUS "MODMESH_PROFILE: ${MODMESH_PROFILE}")
-if(MODMESH_PROFILE)
+option(MM_PROFILE "enable profiler" OFF)
```

Comment:
Don't change it.  Keep it `MODMESH_PROFILE`.

Renaming does not do any good, so don't rename.  Contributors and maintainers have been familiar with the old name.  Don't destroy memory.

Keep in mind that we always refactor when we want to speed up.  This renaming cannot have anything to do with speedup.